src/main/java/com/shapesecurity/salvation2/Salvation2.java

```diff
@@ -0,0 +1,52 @@
+package com.shapesecurity.salvation2;
+
+import org.teavm.jso.JSBody;
+
+public class Salvation2 {
```

Comment:
Since this is only for use in the TeaVM build, maybe call this "JsInterface" or something?

Reply:
Works for me